tests/test-demo-buildsystem.sh

```diff
@@ -19,9 +19,12 @@
 
 set -euo pipefail
 
-if ! fusermount --version >/dev/null 2>&1; then
+if ! which fusermount >/dev/null 2>&1; then
     echo "1..0 # SKIP no fusermount"
     exit 0
+elif ! capsh --print | grep ^Current | grep -q cap_sys_admin; then
```

Comment:
I'm not sure this is correct? On my Debian system, I don't have cap_sys_admin in Current, but fusermount is setuid root, so this test doesn't need to be skipped (and indeed it passes).

Perhaps something like (pseudocode)

we have CAP_SYS_ADMIN || (fusermount is setuid root && securebits do not prevent us from taking advantage of that)